### ISSUE TITLE
No types.ts for useragentRegexp and browsers

### ISSUE THREAD

**Issue by jjelev** [browserslist/browserslist-useragent-regexp#330]
**types.ts** are not generated for **useragentRegexp** and **browsers**.
The files are missing from build but are present in the source.
I tested on the latest version 2.0.0 also on 1.3.1 and the results are the same.

```javascript
import { getUserAgentRegExp } from 'browserslist-useragent-regexp'
const brRegex = getUserAgentRegExp({ allowHigherVersions: true })
```
This will result with an error when you I am trying to build TypeScript project.

**Comment by dangreen** [comment]
@jjelev Hi! Thank you for this issue.

Fixed and published: v2.0.1

https://unpkg.com/browse/browserslist-useragent-regexp@2.0.1/lib/browsers/
https://unpkg.com/browse/browserslist-useragent-regexp@2.0.1/lib/useragentRegexp/

**Comment by jjelev** [comment]
Very appreciated. Fantastic work!
